src/constants.js

```diff
@@ -348,3 +348,7 @@ export const USER_TYPES = {
   SYSTEM: 'system',
   DCB: 'dcb',
 };
+
+export const DCB_USER = {
+  lastName: 'DcbSystem'
```

Comment:
Can we use this function to identify DCB users, instead of a hard-coded last name?
https://github.com/folio-org/ui-users/blob/master/src/components/util/util.js#L205

If not, maybe by a constant `id` if one exists? I'm just thinking that it might be possible to create multiple users with this last name

Reply:
Thank you! I have refined the logic accordingly. Also, cleaned up the new files.